modules/integration/src/test/java/org/apache/fluo/integration/TestTransaction.java

```diff
@@ -137,11 +137,13 @@ public boolean preCommit(CommitData cd, RowColumn primary) {
     return tx.preCommit(cd, primary);
   }
 
+  // TODO - do we still want to test like this with the new format?
```

Comment:
We still want to offer this functionality for testing.  Were you thinking of offering it in a different way?

Are there any other new TODOs you created that I have not addressed?

Reply:
This was before I had changed the testing methods to behave the way they do now. These Todo's aren't relevant anymore, I've removed them.